### PR TITLE
Add auto_increment column metrics

### PR DESCRIPTION
**Pull request by SuperQ**
Add metrics for reporting the current value of auto_increment columns

Based on the reporting query:
https://github.com/SuperQ/bk-tools/blob/master/pk-full-ratio.sql

### CODE REVIEW

**Comment by SuperQ** [comment]
@matthiasr This adds the current value and max metrics for `auto_increment` columns.

**Comment by SuperQ** [comment]
@ebartels This will let us use native prometheus reporting, and the new magic "predict" functionality to just give us an estimated date in the future when things will fail.

File: mysqld_exporter.go [hunk 1]
```diff
@@ -49,9 +53,27 @@ const (
...
 	globalStatus      = "global_status"
+	informationSchema = "info_schema"
 	performanceSchema = "perf_schema"
+	slaveStatus       = "slave_status"
+)
+
+// Metric SQL Queries.
+const (
+	infoSchemaAutoIncrementQuery = `
+SELECT table_schema, table_name, column_name, auto_increment,
```
Comment (juliusv) [inline mysqld_exporter.go]: Why not indent one tab more than the `infoSchemaAutoIncrementQuery`? Makes indentation levels visually clearer.

**Comment by juliusv** [comment]
:+1: otherwise on the Go side, can't judge the MySQLy bits too much.

**Comment by ebartels** [comment]
:+1:

**Pull request merged**